config.yaml

```diff
@@ -237,6 +237,23 @@ options:
       integration bridge on the hypervisor and during a migration the operator
       may want to shut down and clean up after the ML2 OVS components before
       the `ovn-controller` takes over and reprograms the bridge flow rules.
+  prefer-chassis-as-gw:
+    type: boolean
+    default: false
+    description: |
+      Prefer units of the named application in CMS scheduling of HA chassis
```

Comment:
After the chat we had offline, I suggest this, but feel free not to take it if that doesn't make sense to you:

```
      Prefer units of this application in CMS (Cloud Management System)
      scheduling of HA chassis groups (aka. gateways) over units of other OVN
      chassis applications present in this deployment.
```

Then I'd keep all your 3 other paragraphs as is. And I would add one more note:

```
      .
      NOTE: It may also make sense to enable this option on several OVN chassis
      applications at the same time, e.g. on 2 out of 3.
```

Reply:
Excellent suggestion :+1: